scripts/docker/zeppelin/bin/env_python_3_with_R.yml

```diff
@@ -17,7 +17,7 @@ dependencies:
   - jupyter_client=5
   - bokeh=1.3.4
   - panel
-  - holoviews
+  - holoviz=0.11.6
```

Comment:
Very interesting package. But maybe we should give users the possibility to build the Docker image with the latest version of holoviz without changing the version in the Dockerfile.
Since holoviz is a meta package, it might make sense to remove the [dependencies](https://github.com/holoviz/holoviz/blob/master/setup.cfg#L33-L50). e.g. numpy, bokeh, matplotlib
A small comment for this particular dependency should also be placed.

Reply:
@Reamer  sorry for late response, it seems adding holoviz sometimes doesn't work, I have removed it and add holoviews